OAuthSwift/OAuthSwiftClient.swift

```diff
@@ -10,9 +10,10 @@ import Foundation
 
 var OAuthSwiftDataEncoding: NSStringEncoding = NSUTF8StringEncoding
 
-public class OAuthSwiftClient {
+public class OAuthSwiftClient: NSObject {
```

Comment:
I have added NSObject subclassing to allow OAuthSwiftClient to be called from Objective-C. Most other classes inherit from NSObject (in particular OAuthSwift, OAuthSwiftCredential and OAuthSwiftHTTPRequest), so I have thought that it would not be a non sense to do that.